### ISSUE TITLE
having a terminal to execute commands

### ISSUE THREAD

**Issue by maks** [maks/MGit#30]
_From @valluminarias on June 20, 2016 11:42_

Having git version control in mobile such as android is awesome. 
Maybe it's nice that we can execute commands in terminal way.

_Copied from original issue: sheimi/SGit#200_

**Comment by maks** [comment]
yes, having a cli interface could be useful for devices with physical keyboards such as the coming support on Chromebooks. But this is a large feature and not a high priority for now.

**Comment by kaeptmblaubaer1000** [comment]
Maybe this could start as a new menu command in RepositoryDetailActivity to just execute one command using `org.eclipse.jgit.pgm` in the repo directory with just the output displayed? That could also be a workaround for features which don't have a GUI yet.

**Comment by maks** [comment]
My preference would be to actually incorporate a terminal to execute commands and as you point out this would be useful for functionality that is not yet implemented in MGits UI proper. I think having a single command executed at a time with just output would not be a great UX.
